TextAnalytics/pitchfork_data/pitchfork_classification_script.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as lite\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from pandas.io.sql import read_sql\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from fastai.text import *\n",
+    "bs=32"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/media/jlealtru/data_files/github/Tutorials/TextAnalytics/pitchfork_data\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(os.getcwd())\n",
+    "path='/media/jlealtru/data_files/github/Tutorials/TextAnalytics/pitchfork_data'"
```

Comment:
Use relative paths - add either a standalone script that secures data from source or run it in an intro section, but show how to download the source data directly so all your steps can be rebuilt.